array-methods/map.js

```diff
@@ -0,0 +1,15 @@
+const prices = [42.42, 10, 28.2234, 3.2, 5, 12];
+
+const sale = prices.map((el) => {
+  const obj = {
+    price: el,
+    salePrice: el / 2,
+  };
+  return obj;
+});
```

Comment:
It's simpler to write this with the shorthand syntax:
```
const sale = prices.map((el) => ({ prices: el, salePrice: el / 2 }));
```
Remember the quiz question on arrow functions about how to return an object literal?